src/particles/plasma/PlasmaParticleContainer.cpp

```diff
@@ -467,12 +467,11 @@ IonizationModule (const int lev,
             const amrex::ParticleReal Ep = std::sqrt( Exp*Exp + Eyp*Eyp + Ezp*Ezp )*E0;
 
             // Compute probability of ionization p
-            const amrex::Real gammap = (1.0_rt + uxp[ip] * uxp[ip]
-                                               + uyp[ip] * uyp[ip]
-                                               + psip[ip]* psip[ip] ) / ( 2.0_rt * psip[ip] );
+            const amrex::Real gamma_psi = plasma_gamma_psi(uxp[ip], uyp[ip], 1._rt / psip[ip],
+                                                           /* Assumes Aabssq == 0 */ 0._rt);
```

Comment:
Here, for all LPA cases I think it's OK, the ions are heavy and their motion should not be affected by the laser too much anyway.